test/smoke/packages/knative-serving/0.22.0/knative-serving-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Checking package is installed or not
+
+tanzu package installed list | grep "contour.community.tanzu.vmware.com" || {
+    version=$(tanzu package available list contour.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+    tanzu package install contour --package-name contour.community.tanzu.vmware.com -f "${MY_DIR}"/../../contour/1.18.2/contour-values.yaml --version "${version}"
+}
+
+
+tanzu package installed list | grep "knative-serving.community.tanzu.vmware.com" || {
+    version=$(tanzu package available list knative-serving.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+    tanzu package install knative-serving --package-name knative-serving.community.tanzu.vmware.com -f "${MY_DIR}"/serving-values.yaml --version "${version}"
+}
+
+NAMESPACE_SUFFIX=${RANDOM}
+NAMESPACE="knative-serving-${NAMESPACE_SUFFIX}"
+kubectl create ns ${NAMESPACE}
+function cleanup {
+    kubectl delete ns ${NAMESPACE}  
+    tanzu package installed delete knative-serving -y
+    tanzu package installed delete contour -y
+}
+
+kubectl apply -n ${NAMESPACE} --filename "${MY_DIR}"/knative-service.yaml
+echo "Waiting for pod to get scale down..."
```

Comment:
this only tests scale down, we can also test accessing the kscv

Reply:
Agree will check for that too.